components/icons.tsx

```diff
@@ -1,4 +1,4 @@
-import {LogIn, Moon, SunMedium, LucideProps} from "lucide-react"
+import {LogIn, Moon, SunMedium, LucideProps, ArrowLeft} from "lucide-react"
```

Comment:
@tranduybau I think we need to separate 2 kind of icons (from svg and lucide-react). I wrote component Icon in branch `task/review-code-page`
how to use that component
`
<Icon
          name="arrow-left" // auto suggestion all of lucide icon name here
          className="absolute inset-y-0 left-3 mt-2 h-6 w-6 text-gray-500"
 />
 `

Reply:
We will add eslint